packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -38,15 +39,13 @@ import { participantListContainerPadding } from '../common/styles/ParticipantCon
 import { ChatScreenPeoplePane } from './ChatScreenPeoplePane';
 import { toFlatCommunicationIdentifier } from '@internal/acs-ui-common';
 /* @conditional-compile-remove(file-sharing) */
-import { FileDownloadCards } from './FileDownloadCards';
-/* @conditional-compile-remove(file-sharing) */
 import { fileUploadsSelector } from './selectors/fileUploadsSelector';
 /* @conditional-compile-remove(file-sharing) */
 import { useSelector } from './hooks/useSelector';
 /* @conditional-compile-remove(file-sharing) */
-import { useState } from 'react';
```

Comment:
we remove this import and then we are using it directly?

Reply:
To reduce the surface of conditionally compiled code.
Since `React` is already in scope. importing `useState` separately means that we now have to conditionally remove it.
`React.useState` saves us that trouble